pkg/broker/ingress/multi_topic_decouple_sink.go

```diff
@@ -90,6 +97,26 @@ func (m *multiTopicDecoupleSink) Send(ctx context.Context, broker types.Namespac
 	return err
 }
 
+// EventFilterFunc is used to see if a target is interested in an event.
+// It is used as a vaiable to allow stubbing out in unit tests.
+var EventFilterFunc = filter.PassFilter
+
+// hasTrigger checks given event against all targets to see if it will pass any of their filters.
+// If one is fouund, hasTrigger returns true.
+func (m *multiTopicDecoupleSink) hasTrigger(ctx context.Context, event *cev2.Event) bool {
+	var hasTrigger bool
```

Comment:
IMO it will be more readable to have `hasTrigger := false` instead.

Reply:
In go all variables are zero initialized: https://golang.org/ref/spec#The_zero_value.

Reply:
Yes, I do know that, that's why I said readable not bug :). IMO [short variable declaration](https://golang.org/ref/spec#Short_variable_declarations) is a cleaner and easier to understand syntax, especially for these primary types.

Reply:
I agree with @zhongduo, explicitly setting to false is quicker to understand.